protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py

```diff
@@ -0,0 +1,175 @@
+import math
+from opentrons.types import Point
+
+metadata = {
+        'apiLevel': '2.5',
+        'protocolName': 'ThermoFisher MagMAX Plant DNA Isolation',
+        'author': 'Chaz <chaz@opentrons.com>',
+        'source': 'Custom Protocol Request'
+        }
+
+
+def run(ctx):
+
```

Comment:
@Koeng101 there should be a line here that looks like this:
```
[num_samples] = get_values(  # noqa: F821
'num_samples')
```
the variable `num_samples` can then be accessed and will be an integer based on your fields.json

Reply:
Gotcha, another push going with that